orb-ui/src/engine/pearl.rs

```
@@ -207,7 +207,7 @@ impl EventHandler for Runner<PEARL_RING_LED_COUNT, PEARL_CENTER_LED_COUNT> {
             Event::BootComplete { api_mode } => {
                 self.sound
                     .queue(sound::Type::Melody(sound::Melody::BootUp), None)?;
-                self.operator_pulse.stop();
+                let _ = self.operator_pulse.stop(Transition::PlayOnce);
```

Add comment why we discard error